client/js/templates/EntriesPage.jsx

```diff
@@ -144,18 +144,13 @@ function reloadList({
  * @param {number} selected
  */
 function openSelectedArticle(selected) {
-    const link = document.querySelector(`.entry[data-entry-id="${selected}"] .entry-datetime`);
+    const link = document.querySelector(`.entry[data-entry-id="${selected}"] .entry-newwindow`);
     if (selfoss.config.openInBackgroundTab) {
         // In Chromium, this will just cause the tab to open in the foreground.
         // Appears to be disallowed by the pop-under prevention:
         // https://crbug.com/431335
         // https://crbug.com/487919
-        const event = new MouseEvent(
-            'click',
-            {
-                ctrlKey: true,
-            }
-        );
+        const event = new MouseEvent('click', { ctrlKey: true, bubbles: true});
```

Comment:
Why bubble?

Reply:
The event handler for marking the entry as read doesn't fire otherwise. I'm not 100% sure why, because the event is dispatched on the element we registered the handler to, but without bubbling only the link opens but the handler doesn't run. Anyway, `.click()` and actual clicks bubble by default too, so we just make it behave the same.